app/assets/stylesheets/views/_taxons.scss

```diff
@@ -54,11 +54,15 @@
 
       li {
         @include grid-column(1 / 3);
-        padding-bottom: $gutter;
+        padding-bottom: $gutter * 1.5;
```

Comment:
I was trying to make it clearer where the difference would be (mobile only), but that's perhaps too verbose. Will make the change.